### DISCUSSION TITLE
Requirements for mobile compatibility

### DISCUSSION THREAD

**Discussion by fetwar** [mvdkwast/obsidian-copy-as-html#56] [General]
Firstly, thanks for making such a nice plugin. this plugin is really well done and certainly seems more featureful that the other copy as html Obsidian plugin.

The only drawback I can see with this plugin is that it doesn't have mobile support, which makes me wonder, what would be the requirements to allow mobile compatibility?

Are there any particular blocking issues that prevent mobile compatibility from being added?

Thanks, if it isn't too difficult I would love to add mobile support for this plugin

**Comment by fetwar** [comment]
@mvdkwast :)

**Reply by mvdkwast** [reply]
Here you go : #61 :)

I honestly don't remember why it didn't work when I first implemented the plugin, but I didn't investigate since I didn't really see a use for it. But being the most requested feature, it seems that others do !

**Comment by mvdkwast** [comment]
Unfortunately this will not be possible, at least on Android. More details here: #61
